bin/pyrdp-mitm.py

```diff
@@ -43,6 +44,9 @@ def main():
 
     logger.info("MITM Server listening on port %(port)d", {"port": config.listenPort})
 
+    if "HOST_IP" in os.environ:
+        logger.info("Host IP: %(host_ip)s", {"host_ip": os.environ["HOST_IP"]})
```

Comment:
I think both messages should be in the same log. This will greatly simplify event processing later for a little cost.

Reply:
sure